superset/queries/saved_queries/commands/importers/v1/__init__.py

```diff
@@ -58,7 +61,12 @@ def _import(
         database_ids: Dict[str, int] = {}
         for file_name, config in configs.items():
             if file_name.startswith("databases/") and config["uuid"] in database_uuids:
-                database = import_database(session, config, overwrite=False)
+                database = import_database(
+                    session,
+                    config,
+                    overwrite=False,
+                    requester_as_owner=requester_as_owner,
```

Comment:
Seems like most of the work in this PR is carrying this parameter through the stack. I think a simpler solution might be setting the owner attribute directly in the configs, so that we don't have to carry this boolean.